### ISSUE TITLE
Get the initialization neural network working

### ISSUE THREAD

**Issue by eigenvivek** [eigenvivek/DiffDRR#43]
Nalini's advice:

1. Check the training loss
2. Make sure the network can learn a single example
3. Use batchnorm
4. Use residual connections
5. Try a deeper network
6. Try to predict just a few parameters (eg, just project through an axis-aligned slice)

**Comment by eigenvivek** [comment]
Batched DRR rendering was completely broken (#107), but fixed now!
